snippets/csharp/VS_Snippets_VBCSharp/csProgGuideEnums/CS/Enums.cs

```diff
@@ -14,14 +14,14 @@ enum Month : byte { Jan, Feb, Mar, Apr, May, Jun, Jul, Aug, Sep, Oct, Nov, Dec }
     [Flags]
     enum Days
     {
-        None = 0x0,
-        Sunday = 0x1,
-        Monday = 0x2,
-        Tuesday = 0x4,
-        Wednesday = 0x8,
-        Thursday = 0x10,
-        Friday = 0x20,
-        Saturday = 0x40
+        None        = 0,
```

Comment:
I’d suggest this to make it clear what’s happening, and provide some learning to readers that aren’t familiar with binary, hex and decimal. (Historical note: binary literals were added in part for teaching these constructs.)

```csharp
[Flags]
enum Days
{
    None      = 0b_0000_0000, // 0
    Sunday    = 0b_0000_0001, // 1
    Monday    = 0b_0000_0010, // 2
    Tuesday   = 0b_0000_0100, // 4
    Wednesday = 0b_0000_1000, // 8
    Thursday  = 0b_0001_0000, // 16
    Friday    = 0b_0010_0000, // 32
    Saturday  = 0b_0100_0000  // 64 
}
```

Reply:
That, I like.